modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -53,6 +52,7 @@ export type Tileset3DProps = {
   loadTiles?: boolean;
   basePath?: string;
   maximumMemoryUsage?: number;
+  memoryCacheOverflow?: number;
```

Comment:
Does this differs from other cache management options. Or was it just not plumbed through? Why "overflow" instead of "size" or "MB"?

Nit: add a one-line TSDoc comment before each row summarizing function? 
```suggestion
  /** ... */
  memoryCacheOverflow?: number;
```

Reply:
> I tried to explain this in the PR.

Yes thanks. I personally do like to have comments in the code as well, as PRs are ephemeral and I may not remember this in a year when we come back to this.

--- 

As a side not it looks like the documentation on tileset memory management is in somewhat poor shape. Beyond being somewhat hard to find, https://loaders.gl/docs/modules/tiles/api-reference/tileset-3d#maximummemoryusage--number, there are also formatting issues and some unclarities.

Assuming the `maximumMemoryUsage` is the hard limit on how much memory a tileset is ever allowed to load (including visible tiles and cache), then maximumMemoryOverflow must always be a smaller value? And if it isn't then we can't really apply any hard limits to protect mobile devices, can we?

Reply:
The hard limit is actually `maximumMemoryUsage  + maximumMemoryOverflow`.  I can see how the use of the word "maximum" here creates a confusion. In Cesium they are just called `cacheBytes` and `maximumCacheOverflowBytes`. I would maybe change `maximumMemoryUsage` to `memorySize`? I was avoiding to change that name to not break compatibility. But I guess this could be solved by keeping the old name and emitting a deprecation warning?

Reply:
I think this would require some bike-shedding. For now I agree with your approach of avoiding any breaking changes. So, your current approach is fine!